csharp/tests/ParallelSelectExtTest.cs

```diff
@@ -154,12 +154,12 @@ private static IEnumerable ParallelSelectLimitCases()
                           useAsync,
                           2,
                           20);
-        yield return Case(blocking,
-                          useAsync,
-                          10,
-                          100);
         if (!blocking)
         {
+          yield return Case(blocking,
```

Comment:
This should come from another PR and merge or rebase

Reply:
Noted !

Reply:
I'm gonna merge it